### ISSUE TITLE
add additional features to PDB parser and make them options

### ISSUE THREAD

**Issue by psifertex** [Vector35/binaryninja-api#695]
- [x] apply vtables
- [ ] apply local variables 
- [ ] support bitwise structures (depends on #694)
- [ ] apply register variables
- [x] use new debuginfo APIs for better performance and accuracy (#2263)

**Comment by CouleeApps** [comment]
> apply vtables
> use new debuginfo APIs for better performance and accuracy (https://github.com/Vector35/binaryninja-api/issues/2263)
Done

> apply local variables
> apply register variables
These require us to support loading local variables from DebugInfo. I'm opening a separate issue for that, since it's an entire ordeal.

> support bitwise structures (depends on
https://github.com/Vector35/binaryninja-api/issues/694)
Preliminary support is done for this, but the real fix is blocking on that issue. For now, bitwise structures will create unions of integers with all the relevant members.

**Comment by emesare** [comment]
Support for bitwise structures was added in 5.2
